README.md

```diff
@@ -97,17 +97,93 @@ These new flags will be only useful until a certain threshold of monitored valid
 
 ### validators
 
-This flag is self-explanatory in general but you need to respect the following separation rules:
+#### Accepted formats
+
+The following formats are accepted:
+
+* validator index e.g. 123456
+* validator pubkey e.g. 0xaffc434cf8138634a4cd0ef6cb815febd3db25760b1b6c522f9b4aa78e599b60336d7dd2e953192e45d4ac91f66f0723
+* validator index/pubkey with alias e.g. "123456;my-validator"
+
+#### Separation
+
+The following separation rules apply while adding multiple identifiers:
 
 * Validator identifiers can be separated by comma or space e.g.:
   * `--validators 123 456 --validators 678 999`
   * `--validators 123,456 --validators 678,999`
 * If you provide a validator identifier with an alias you need to wrap the whole string of one identifier-alias-pair in quotes or double quotes e.g.:
   * `--validators "123;val1" "456;val2" --validators 678 999` or
-  * `--validators "123;val1","456;val2" --validators 678,999`
+  * `--validators '123;val1','456;val2' --validators 678,999`
 
 **Note: Wrapping an identifier-alias-pair in quotes or double quotes is not true for a `docker-compose`. Please check the example [compose](docker/compose.yaml).**
 
+### rest
```

Comment:
should be called API or server or RESTful API as rest is just the technique behind the actual feature